### DISCUSSION TITLE
Defining variable polling interval

### DISCUSSION THREAD

**Discussion by Nardol** [containrrr/watchtower#951] [Q&A]
Hello,

Is it possible to define a variable interval for polling?
I looked on schedule side and --interval without success.

I would like to set a polling interval between 36 et 72 hours.

Thanks.

**Comment by zoispag** [comment]
Hi Nardol,

You can find the instructions for the polling interval setting [here](https://containrrr.dev/watchtower/arguments/#poll_interval).
Can you share your watchtower configuration, to try and understand why it's not working for you?

**Reply by Nardol** [reply]
For the moment, I have not defined any value so I have the expected default behavior.
But reading the documentation you suggested me, I understand how to set a fixed interval, for example 129600 seconds.
`-i 129600` would be OK.
But Can I do `--interval 129600-221600` for example?
Or putting two `--interval` in the `docker run` command?
As it is not documented, to be honest I have not tested :slightly_smiling_face: or did I miss something?

**Reply by zoispag** [reply]
Flag `--interval` accepts one number; that is how often (in seconds) watchtower will look for updates.
What are you trying to achieve with the 2 numbers?
Setting up `--interval 129600` would result in watchtower checking every 36 hours for an updated image to pull.

**Reply by Nardol** [reply]
I operate a Storj Node which is running in a docker container.
Storj provides their fork of Watchtower, which implement an hard-coded checking interval of 36-72H, so a random interval to be sure all nodes won't be updated at the same time.

The problem with their fork is they don't support the `--scope` flag, which I need as I have other containers I would like to update more frequently.
When I create the container with their fork, I pass the two containers (storagenode watchtower) in parameter. But no scope so as the running multiple instance documentation specifies, because there is no scope my other watchtower container could be cleaned up. right?
My second Watchtower, not based on the fork, is not named watchtower but watchtower-original.

So the best solution looks like not to use their fork but to do so, I wish to keep their update polling interval so not a fixed one but a variable interval between 36 and 72 hours.

I hop I explained right my use case.

**Reply by simonaronsson** [reply]
Thank you for the detailed explanation! Variable intervals are unfortunately not supported at the moment. 

If you really think the variation is essential, I would recommend you to schedule a cronjob running watchtower with the `--run-once` argument.
